docs/general/attributes.md

```diff
@@ -209,6 +142,48 @@ if they do not cause breaking changes to HTTP semantic conventions.
 | `ipv6` | IPv6 |
 <!-- endsemconv -->
 
+### `network.peer.*` and `network.local.*` attributes
+
+These attributes identify network peers that are directly connected to each other.
+
+For IP-based communication, `network.peer.address` and `network.local.address` should be IP addresses, Unix domain names, or other addresses specific to network type.
+
+Semantic conventions and instrumentations that populate `address`/`port` pairs
+on `server.*`, `client.*`, `destination.*` or `source.*`, SHOULD only populate the `address`/`port` pair on
+`network.peer.*` and `network.local.*` when they are not the same.
```

Comment:
I disagree with not populating. Then you'll need a database of instrumentation library version/name to find if absence means "not implemented" or "same".

Reply:
discussed with @lmolkova and removed this sentence and the `network.peer.address` condition in `model/trace/http.yaml`:

> ```
> requirement_level:
>   recommended: If different than `client.address`.
> ```